src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -214,6 +190,24 @@ class HeaderUI extends Component<Props, State> {
     const leftNavControls = navControls.bySide[NavControlSide.Left];
     const rightNavControls = navControls.bySide[NavControlSide.Right];
 
+    let navLinksArray = navLinks.map(navLink =>
+      navLink.hidden
+        ? null
+        : {
+            key: navLink.id,
+            label: navLink.title,
+            href: navLink.href,
+            iconType: navLink.euiIconType,
+            size: 's',
+            style: { color: 'inherit' },
+            'aria-label': navLink.title,
```

Comment:
Do we need `aria-label`'s when they're the same thing as the actual button text?

Reply:
What about in the case where the drawer is collapsed? Visually you only see the icons and not the button label. Given that it is visually hidden, I think we want keep the `aria-label`, right?

Reply:
The screen reader actually still does read the button label because it's visually hidden not `display: none`.

I removed it from the recently viewed and you can see that it's still reading the button label, while the "favorites" reads both the label and aria-label.

<img src="https://d.pr/free/i/S1yPMN+" width="50%" />

Reply:
Right, but I think `aria-label` is also recommended when the text is not visible on the screen. This is the reference I'm using: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
That said, one could make the argument that the tooltips solve that problem of it being visible when in the collapsed state.

Reply:
It's probably safe to keep in, though usually aria attributes are for assistive tech (like screenreaders) so I *think* when they say "not visible" it's really that the text doesn't exist (using an icon instead of text) or `display: none` which assistive tech then ignores.